packages/friends/yml/en/isaac-penington.yml

```diff
@@ -208,6 +235,10 @@ documents:
       by an insatiable hunger for truth, he sought the Lord with all his heart and
       discovered a Christianity that stood in, and flowed out from, the light and life of
       Jesus Christ reigning in the inner man.
+    partial_description:
+      Volume Two of The Writings of Isaac Penington, a conscientiously abridged, and
+      slightly modernized and annotated edition of his four volume complete works (which
+      can also be found on this site.)
```

Comment:
Should these two editions say the part about "can also be found on this site"? Because at least in the beginning, they can't. 🤔  - we have the books listed in the yml, but they are marked as `draft` - so they won't appear on the website, since we don't actually have asciidoc for them yet, and I'm not sure we sorted out how we felt about making them available while QHP still exists.